Dockerfile

```diff
@@ -18,7 +21,19 @@ RUN /bin/sh -c set -eux; apk add --update \
     openjdk11 \
     fontconfig \
     ttf-dejavu \
-    && rm -rf /var/cache/apk/*
+    icu-libs \
+    krb5-libs \
+    libgcc \
+    libintl \
+    libintl \
+    libintl \
```

Comment:
This seems repeated

Reply:
You are a lot more alert than I am today Vadim :) 
Cosmetic defect that I'll clean up next time I need to change the project. I need to correct the bush of build tags soon anyways.